packages/cognito-custom-mail-lambda/src/mailer/templates/admin-user-invite.mjml

```diff
@@ -30,11 +30,11 @@
         <mj-text mj-class="body-text">Hi {name},</mj-text>
         <mj-text mj-class="body-text">Welcome to Reapit Connect.</mj-text>
         <mj-text mj-class="body-text">Reapit Connect is our single sign on solution which allows you to seamlessly access products and services provided by Reapit Ltd.</mj-text>
-        <mj-text mj-class="body-text">Please see below your userName and temporary password. When you click the login link, you will be re-directed to a screen where you will be asked to change your password.</mj-text>
+        <mj-text mj-class="body-text">Please see below your userName and temporary password. When you click the login link, you will be re-directed to a screen where you will be asked to change your temporary password.</mj-text>
```

Comment:
Slightly against the ticket, but reading this doesn't read brilliantly as we are using the word temporary twice in the same sentence. Please can you just revert this change

Reply:
Updated the text